Lectures/MyUtils/src/clang.jl

```diff
@@ -87,12 +88,20 @@ function compile(
         push!(args, "-I$(dir)/include")
         push!(args, "-L$(dir)/lib")
     end
+    cmp = compiler(code, mpi)
+    if mpi && !use_system
+        dir = dirname(dirname(MPI.API.libmpi))
+        cmp = joinpath(dir, "bin", "mpicc")
+        push!(args, "-I$(dir)/include")
```

Comment:
`mpicc` has the prefix configure to `/workspace/...` so I think it's not built correctly on Yggdrasil so I need to add these manually